packages/react-native/src/primitives/Radio/Radio.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useMemo } from 'react';
+import { Pressable, View, ViewStyle } from 'react-native';
+
+import { Label } from '../index';
+
+import { RadioProps } from './types';
+import { styles } from './styles';
+
+export default function Radio<T>({
+  buttonStyle,
+  disabled,
+  label,
+  labelPosition = 'end',
+  labelStyle,
+  onChange,
+  selected,
+  size,
+  style,
+  value,
+  ...rest
+}: RadioProps<T>): JSX.Element {
+  const handleOnChange = useCallback(() => {
+    if (!disabled) {
+      onChange?.(value);
+    }
+  }, [onChange, value, disabled]);
+
+  const containerDirection: ViewStyle = useMemo(
```

Comment:
Can we consolidate all the container style logic here and rename as `containerStyle`?